src/admin/components/forms/DraggableSection/ActionPanel/types.ts

```diff
@@ -1,7 +1,7 @@
 import { Block } from '../../../../../fields/config/types';
 
 export type Props = {
-  label: string
+  label: string | false
```

Comment:
I believe this should always have a label as well. Within the `ActionPanel`, we use labels for "Add Row" and "Remove Row" / etc. It should be sanitized at this point I believe.